src/geode/mesh/builder/solid_mesh_builder.cpp

```diff
@@ -339,69 +433,8 @@ namespace geode
         if( solid_mesh_.are_edges_enabled()
             || solid_mesh_.are_facets_enabled() )
         {
-            for( const auto f : LRange{ solid_mesh_.nb_polyhedron_facets(
```

Comment:
Ok the skip is here? What is the best place?

Reply:
I like it there